rplugin/python3/denite/scantree.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import os
+
+
+def scantree(pathn):
+    """Use scandir if available else os.walk.
+    scandir is supposed to be much faster"""
+
+    skip_list = ['.git']
+
+    if 'scandir' in dir(os):
```

Comment:
I will follow your suggestion but right now the script is called every time with python so that only gets executed once.

Maybe in the future it can be imported and used directly and in that case it makes sense to care a little more about the performance.